src/main/java/org/sasanlabs/service/vulnerability/xss/reflected/XSSWithHtmlTagInjection.java

```diff
@@ -85,4 +85,70 @@ public ResponseEntity<String> getVulnerablePayloadLevel3(
         }
         return new ResponseEntity<String>(payload.toString(), HttpStatus.OK);
     }
+
+    // Just adding User defined input(Untrusted Data) into div tag if doesn't contains
+    // eval(...) expression which evaluates the string expression and returns its value.
+    // Can be broken by various ways
+
+    @AttackVector(
+            vulnerabilityExposed = VulnerabilityType.REFLECTED_XSS,
+            description = "XSS_DIRECT_INPUT_DIV_TAG_AFTER_REMOVING_VALUES_CONTAINING_EVAL_EXPRESSION")
+    @VulnerableAppRequestMapping(value = LevelConstants.LEVEL_8, htmlTemplate = "LEVEL_1/XSS")
+    public ResponseEntity<String> getVulnerablePayloadLevel8(
+            @RequestParam Map<String, String> queryParams) {
+        String vulnerablePayloadWithPlaceHolder = "<div>%s<div>";
+        StringBuilder payload = new StringBuilder();
+        Pattern pattern = Pattern.compile("eval\\((.*?)\\)");
```

Comment:
can you please add an extra blacklist defence to level 3 here?

Reply:
or make it as level 2 as first level has not defence and level 2 has eval based defence and then move other levels to next levels like level3 becomes level 4 and so on.